src/helpers/airdrop.ts

```diff
@@ -1,7 +1,86 @@
-import { CONST } from 'app/consts';
+import { CONST, AirdropGoal, AirdropCarouselCardData } from 'app/consts';
 import { formatDate, getTimezoneOffset, isAfter } from 'app/helpers/date';
 
+import { formatToBtcvWithoutSign } from '../../utils/bitcoin';
+
+const i18n = require('../../loc');
+
 export const getFormattedAirdropDate = () =>
   `${formatDate(CONST.airdropDate.local(), 'DD/MM/YYYY h a')} ${getTimezoneOffset()}`;
 
 export const isAfterAirdrop = () => isAfter(new Date(), CONST.airdropDate);
+
+export const readableOrder = [
```

Comment:
cannot be done like this, if we will change the language the translations will remain from the old language

Reply:
yeah, had that in mind, but took it from email notifications and thought that since it s there and got accepted, it also will work here. What would be good way to share this code then?

Reply:
so someone made mistake there :P maybe const getReadableOrder = () => [i18n.order.first, ....]